awscli/examples/scheduler/get-schedule-group.rst

```diff
@@ -0,0 +1,17 @@
+**To retrieves the specified schedule group**
```

Comment:
```suggestion
**To retrieves the specified schedule group**

```

This is missing a line break and therefore doesn't render correctly.

Reply:
resolved issue. published new commit for examples